scripts/config/msu/utils.nut

```diff
@@ -138,4 +139,15 @@
 		}
 		throw ::MSU.InvalidValue(_compareResult);
 	}
+
+	function getActiveState()
+	{
+		foreach (name, state in this.States)
+		{
+			if (state != null && state.isVisible() && name != "root_state")
```

Comment:
Is that last one necessary since we hookChildren?

Reply:
It is unfortunately